src/WormholeL2ERC20.sol

```diff
@@ -24,6 +24,14 @@ contract WormholeL2ERC20 is ERC20Votes, WormholeReceiver, WormholeSender {
   /// @dev Contract is already initialized with an L2 token.
   error AlreadyInitialized();
 
+  event Unlock(
```

Comment:
I'm questioning the name of this event, which also makes me question the name of the method its emitted in. For the moment it mostly makes sense, but if you think of expanding this to a cross-chain context and it becomes muddier. Maybe it should be something like `BridgedIn` instead?

Should we also have a `BridgedOut` event for when tokens are moved from L1 -> L2? It would include the source chain and address, which are not logged by the `Mint` event, in addition to the receiver and amount (which are logged by `Mint`). Leaning toward

Reply:
I agree the naming should be changed. My suggestion would be `TokenBridged` or something along those lines, and I link both directions L1 -> L2 and L2 -> L1 should have the same event.

Reply:
> I agree the naming should be changed. My suggestion would be TokenBridged or something along those lines, and I link both directions L1 -> L2 and L2 -> L1 should have the same event.

I like this @alexkeating. What do you think @wildmolasses?

Reply:
added!